Source/SpacePort/SpacePort/Menu.cs

```diff
@@ -1,39 +1,255 @@
-﻿using System;
+﻿using Newtonsoft.Json.Linq;
+using System;
 using System.Collections.Generic;
 
 namespace SpacePort
 {
     public class Menu
     {
-        static StarWarsApi api = new StarWarsApi();
-        public static string SignIn { get; set; }
+        static StarWarsApi swAPI = new StarWarsApi();
+        static Spaceport spacePort = new Spaceport();
 
-        public static void DisplayMenu()
+        static JObject jObject = new JObject();
+
+        static string travelerName = "";
+        static string shipName = "";
+        static DateTime arrivalTime;
+        static DateTime departureTime;
+        static int totalCost = 0;
+        static double duration;
+        static bool notValidTime;
+
+        public static int MenuChoice { get; private set; }
+
+        public static void Display()
```

Comment:
You could reduce the size of this method by making each "choice" a method. For example in "while (MenuChoice == 0)", you can make its content to a method.